client/web/src/enterprise/batches/detail/changesets/BatchChangeChangesets.tsx

```diff
@@ -233,6 +239,22 @@ const BatchChangeChangesetsImpl: React.FunctionComponent<Props> = ({
 
     const showSelectRow = viewerCanAdminister && (selected === 'all' || selected.size > 0)
 
+    const renderEmptyElement = useCallback(() => {
```

Comment:
Thoughts on memoizing just the return value, rather than defining this as a callback?

```suggestion
    const emptyElement = useMemo(() => {
```

_(then, further down)_

```diff
- emptyElement={renderEmptyElement()}
+ emptyElement={emptyElement}
```

Reply:
No downsides to this. I'll switch to `useMemo`.